src/extraction/FStar.Extraction.ML.Modul.fs

```diff
@@ -990,6 +1092,14 @@ let extract (g:uenv) (m:modul) =
            BU.measure_execution_time msg (fun () -> extract' g m)
       else extract' g m)
   in
+  let g, mllib =
+    match mllib with
+    | None ->
+      g, mllib
+    | Some mllib ->
+      let g, mllib = UEnv.with_typars_env g (fun e -> RemoveUnusedParameters.elim_mllib e mllib) in
```

Comment:
Am I reading correctly that this is always done, for both F# and OCaml? Wondering if there's any gotcha we should be aware of for OCaml.

Reply:
The branching on FSharp is done in RemoveUnusedParameters..elim_mllib